docs/contributing/overview.stories.mdx

```diff
@@ -0,0 +1,81 @@
+import { Meta } from '../../.storybook/components';
+
+<Meta title="Introduction/Contributing/Overview" />
+
+# Contributing
```

Comment:
Would there be a way to reuse `CONTRIBUTING.md` in the Storybook? We'd have to duplicate any changes to both places

Reply:
That's already happening (see the `copy-docs.sh` script in the `/scripts` folder). I accidentally committed this file because I forgot to add it to the `.gitignore` file, thanks for pointing that out! 🙌🏻